scripts/_common.sh

```diff
@@ -330,16 +313,20 @@ ynh_handle_app_migration ()  {
 #
 # usage: ynh_multimedia_build_main_dir
 ynh_multimedia_build_main_dir () {
-        local ynh_media_release="v1.0"
-        local checksum="4852c8607db820ad51f348da0dcf0c88"
+        local ynh_media_release="v1.2"
+        local checksum="806a827ba1902d6911095602a9221181"
 
         # Download yunohost.multimedia scripts
         wget -nv https://github.com/YunoHost-Apps/yunohost.multimedia/archive/${ynh_media_release}.tar.gz 
 
-        # Verify checksum
+        # Check the control sum
```

Comment:
Wasn't "checksum" more appropriate...? :thinking:

Reply:
Yep maybe, but that would be fixed in the helper itself. That's just a copy of it.